Source/msg.cpp

```diff
@@ -1334,15 +1322,15 @@ void __fastcall NetSendCmdDamage(BOOL bHiPri, unsigned char bPlr, unsigned int d
 		NetSendLoPri((unsigned char *)&cmd, 6u);
 }
 
-void __fastcall NetSendCmdString(int a1, const char *pszStr)
+void __fastcall NetSendCmdString(int player_mask, const char *pszStr)

```

Comment:
I think something like pmask or plrMsk would more closely match existing names.

Reply:
That's not the right function, and this parameter is 32 bits.

Reply:
Of course it's not the right function. The point was that they generally used `pmask` for the variable name of this type.